java/vector/src/main/java/org/apache/arrow/vector/ZeroVector.java

```diff
@@ -109,7 +109,7 @@ public void clear() {
 
   @Override
   public Field getField() {
-    return new Field(name, FieldType.nullable(new Null()), null);
+    return new Field(NAME, FieldType.nullable(new Null()), null);
```

Comment:
why not just change the name to $data$?

Reply:
It would be ideal I think, as in this case we won't need changes to ListVector and FixedSizeListVector at all.
Just may be part for backward compatibility.
I need to verify that "default" is not used for any other purpose though.